auto_submit/lib/service/validation_service.dart

```diff
@@ -146,9 +146,26 @@ class ValidationService {
       }
     }
     // If we got to this point it means we are ready to submit the PR.
-    bool processed = await processMerge(config, result, messagePullRequest);
-    if (processed) await pubsub.acknowledge('auto-submit-queue-sub', ackId);
-    log.info('Ack the processed message : $ackId.');
+    try {
```

Comment:
Nit: move this block logic to a separate sub function for better readibility.

Reply:
Done.